### ISSUE TITLE
Partition of nyud2 dataset

### ISSUE THREAD

**Issue by xiayuwen** [yanx27/3dgnn-pytorch#17]
In the code, why use the same batch of data as both training set and validation set? I divided the training set and test set according to the method in the paper, and mIou decreased a lot, only about 30%. Can someone help me answer my doubts？
